src/julia-config-p.h.in

```diff
@@ -0,0 +1,9 @@
+//
+
+#ifndef JULIA_CONFIG_P_H
+#define JULIA_CONFIG_P_H
+
+#define JL_SYSTEM_IMAGE_PATH "@JL_SYSTEM_IMAGE_PATH@"
```

Comment:
I suppose you're right. I don't think it's worth the clutter of using `configure_file` for this at this point.